### PR TITLE
Remove unused imports in `removeElementAtPath`

### PR DESCRIPTION
**Pull request by liady**
**Problem:**
When inserting a new element using drag&draw - the import for this element remains in `storyboard.js` (even if the element itself is being inserted elsewhere).

**Cause:**
We are not removing unused imports when we call `removeElementAtPath`

**Fix:**
Adding the ability to remove unused imports inside `removeElementAtPath` - if are not used anymore

### CODE REVIEW

**Review by ruggi — APPROVED** [review]

**Review by seanparsons — APPROVED** [review]

File: editor/src/components/editor/import-utils.ts [hunk 1]
```diff
@@ -212,3 +223,54 @@ export function getImportsFor(
 
   return emptyImports()
 }
+
+export function removeUnusedImports(
+  element: JSXElement,
```
Comment (Rheeseyb) [inline editor/src/components/editor/import-utils.ts:228]: Minor point here, but I'd suggest renaming this to `removedElement`, and possibly even renaming the function to reflect that this is removing unused imports specifically for that removed element

**Comment by liady** [comment]
Tests are passing but I want to check some edge case that popped up (with the worker update) before merging.

**Review by Rheeseyb — CHANGES_REQUESTED** [review]
As discussed, this causes a breaking issue when dragging to insert

**Comment by liady** [comment]
> As discussed, this causes a breaking issue when dragging to insert

Fixed (not removing the imports when reparenting the same file)

**Review by balazsbajorics — APPROVED** [review]
